src/mlpack/methods/softmax_regression/softmax_regression.hpp

```diff
@@ -36,27 +36,24 @@ namespace regression {
  *
  * @code
  * arma::mat train_data; // Training data matrix.
```

Comment:
I know you didn't wrote that part but do you mind to change the style to ``trainData``, just to match with the rest of the codebase?